### DISCUSSION TITLE
ZHA error with no message

### DISCUSSION THREAD

**Discussion by qvqn** [zigpy/zha-device-handlers#1911] [General]
Hi,

Im trying to add a new ZHA instance with a tasmota zigbee bridge pro with this guide
https://notenoughtech.com/home-automation/tasmota-on-sonoff-zb-bridge-pro/

but when I go to add it i get an error that doesn't have a message, has anyone seen this before?

![image](https://user-images.githubusercontent.com/14333303/200717358-1bbcbe3d-31b5-4979-bcc5-fbc8eccc1e16.png)
![image](https://user-images.githubusercontent.com/14333303/200717522-6df0f98d-6fe3-45f4-a246-89c8e5e0b5c7.png)
![image](https://user-images.githubusercontent.com/14333303/200717566-06bfd841-4828-42ee-82fa-21ba72ae7454.png)
![image](https://user-images.githubusercontent.com/14333303/200717579-aa9b3e3f-c2a0-4c50-9124-e9c6f87742b2.png)
![image](https://user-images.githubusercontent.com/14333303/200717707-387bb906-9287-461f-b7c7-3d3cc031ceb3.png)

**Comment by Hedda** [comment]
Suggest trying posting to the Home Assistant community forum instead -> https://community.home-assistant.io/c/configuration/zigbee/39

Note not recommended -> https://www.home-assistant.io/integrations/zha#warning-about-wi-fi-based-zigbee-to-serial-bridgesgateways
